lib/max/language/de/default.lang.php

```diff
@@ -895,7 +895,6 @@
 $GLOBALS['strPwdRecWrongId'] = "Falsche ID";
 $GLOBALS['strPwdRecEnterEmail'] = "Geben Sie nachfolgend Ihre eMail Adresse ein";
 $GLOBALS['strPwdRecEnterPassword'] = "Geben Sie nachfolgend Ihr neues Passwort ein";
-$GLOBALS['strPwdRecResetLink'] = "Link zum Passwort zurücksetzen";
```

Comment:
Only "en" needs to be changed. The other language files are generated every week from Crowdin.

Reply:
Resolved.